src/mimefactory.rs

```diff
@@ -347,7 +347,7 @@ impl<'a, 'b> MimeFactory<'a, 'b> {
                         self.context,
                     );
                     let mut lines = raw.lines();
-                    let raw_subject = if let Some(line) = lines.next() {
+                    let raw_subject = raw.lines().next().unwrap_or_default();
                         line
```

Comment:
Applying suggestion on GitHub was not a good idea :)

Reply:
yip, i've seen this coming, but i was unsure if it was just a drawing-problem on github and the subsequent lines are deleted :)